src/components/Metric/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Item, Parameter, Value, Change, Image } from './styled';
+import increment from './increment.svg';
+import decrement from './decrement.svg';
+
+const Metric = ({ parameter, value, change }) => {
+  const changeLogo = +change.slice(0, change.length - 1) > 0 ? 'inc' : 'dec';
```

Comment:
We can use a simple boolean variable here, maybe call this `positive`.

We can change this to test the first character of `change`, if it's a `-` we can set this to true.

Reply:
This just makes it a bit cleaner, as its quite hard to figure out what's happening here.

Try also moving this out to a separate function, exporting it and adding some unit tests for it!